charts/brigade/templates/apiserver/deployment.yaml

```diff
@@ -142,6 +142,30 @@ spec:
           value: {{ .Values.worker.image.pullPolicy }}
         - name: WORKSPACE_STORAGE_CLASS
           value: {{ .Values.worker.workspaceStorageClass }}
+        ports:
+        - name: healthz
+          containerPort: 8080
+        livenessProbe:
+          httpGet:
+            port: healthz
+            path: /healthz
+            scheme: HTTPS
```

Comment:
I think this would be dependent on `.Values.apiserver.tls.enabled`

This applies in some other spots also.